File: 2021-08-17-async-in-spring-application-context-event/action-in-blog-back/src/main/java/blog/in/action/ActionInBlogApplication.java
package blog.in.action;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@EnableAsync
@SpringBootApplication
public class ActionInBlogApplication {

    public static void main(String[] args) {
        SpringApplication.run(ActionInBlogApplication.class, args);
    }
}

File: 2024-03-16-how-to-test-for-cacheable-test/action-in-blog/src/main/kotlin/blog/in/action/ActionInBlogApplication.kt
package blog.`in`.action

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching

@SpringBootApplication
@EnableCaching
class ActionInBlogApplication

fun main(args: Array<String>) {
    runApplication<ActionInBlogApplication>(*args)
}
